private-templates-service/adaptivecards-templating-client/typescript/api.ts

```diff
@@ -14,1477 +14,1476 @@ import localVarRequest = require('request');
 import http = require('http');
 import Promise = require('bluebird');
 
-let defaultBasePath = 'https://adaptivecms.azurewebsites.net';
-
+let defaultBasePath = "http://localhost:5000";
```

Comment:
Does this mean that when running locally, we will once again be hitting our local environments rather than DEV

Reply:
If possible, could you re-configure your linter so that it is easier for reviewers to see exactly what has been changed?

Reply:
> Does this mean that when running locally, we will once again be hitting our local environments rather than DEV

Should we be hitting our local environment or dev when running locally? We can easily do either.

Reply:
local is fine.